### ISSUE TITLE
Mushroom Wars is not playable with SPU LLVM or ASMJIT + audio not work

### ISSUE THREAD

**Issue by spiraluc** [RPCS3/rpcs3#12772]
### Quick summary

Mushroom Wars can go ingame with LLVM or ASMJIT, but after pressing ´´campaign´´ button in menu, the game freezes with the error VM: Access violation writing location 0x0 (unmapped memory).

If I use SPU interpeter (dynamic), the game works. 

Sound is not functional, only occasional crackling. (it doesn't work with xaudio or cubeb)

### Attach a log file

[RPCS3 SPU interpreter.log](https://github.com/RPCS3/rpcs3/files/9739110/RPCS3.SPU.interpreter.log)

[RPCS3 llvm error.log](https://github.com/RPCS3/rpcs3/files/9739115/RPCS3.llvm.error.log)

### System configuration

CPU: Intel core i5 2430M
GPU: NVIDIA GT525M (openGL 4.6 only) (driver v391.35)
RAM: 8GB
WIN 10 (v21H1)

**Comment by Triticum0** [comment]
Having the same issue but changing the Spu Decoder options doesn't fix the crashing for me

**Comment by Triticum0** [comment]
Update @spiraluc 
Turning on Turning on Accurate SPU DMA Seems to fix the issue but the setting is located in the debug tab.
If you still wanted to play the game here a [link](https://wiki.rpcs3.net/index.php?title=Help:Frequently_Asked_Questions) to be able to use the debug tab.

Note: This probably doesn't count as a real fix so don't close the issue. Also, this should be a regression issue as this game was working back in 2019

**Comment by spiraluc** [comment]
@Triticum0 Turning on Accurate SPU DMA helps to make it playable, but the sound still does not work.

**Comment by digant73** [comment]
is that still an issue?

**Comment by spiraluc** [comment]
Yes it is. The sound still doesn't work and now it crashes after pressing the campaign button without displaying an error in vulkan renderer. OpenGL is playable. So nothing has changed yet.

**Comment by alexconsole04** [comment]
Sound works perfectly fine on my end, same for loading campaign. Only using Accurate SPU DMA.

[RPCS3.log.gz](https://github.com/user-attachments/files/23128815/RPCS3.log.gz)

https://youtu.be/LwJRBDL5Hbw

**Comment by alexconsole04** [comment]
If you can, clear all your cache and test again. Perhaps that could fix it.

**Comment by spiraluc** [comment]
I don't know where the problem is, I cleared all caches, but it always crashes after pressing campaign.

The only thing I can think of is that the game is extremely demanding on the CPU and I can't get to 60FPS.

[RPCS3.log](https://github.com/user-attachments/files/23151270/RPCS3.log)

**Comment by alexconsole04** [comment]
> I don't know where the problem is, I cleared all caches, but it always crashes after pressing campaign.
> 
> The only thing I can think of is that the game is extremely demanding on the CPU and I can't get to 60FPS.
> 
> [RPCS3.log](https://github.com/user-attachments/files/23151270/RPCS3.log)

Not entirely sure what's going on your end, could be a bad dump. But your CPU is below recommended specs so I would be expecting performance issues.

**Comment by spiraluc** [comment]
The sound is probably broken due to CPU performance, the game is now working.
